### PR TITLE
Add book tests

### PR DESCRIPTION
**Pull request by holywyvern**
# Summary

- Installed testing gems
- Configured the testing environment
- Added test for Book model
- Added validations into book model

# Trello Card

https://trello.com/c/rX1Jy2DF

### CODE REVIEW

File: training-app/spec/models/user_spec.rb [hunk 1]
```diff
@@ -5,12 +5,9 @@
 describe User do
   describe '.initialize' do
     it 'creates an user with first and last names' do
-      user = User.create(
-        email: 'test@wolox.com.ar',
-        password: '123123123',
-        password_confirmation: '123123123',
-        first_name: 'Test',
-        last_name: 'TestLastName'
+      user = described_class.create(
```
Comment (juanisierra) [inline training-app/spec/models/user_spec.rb:6]: Please leave them in column

**Review by juanisierra — CHANGES_REQUESTED** [review]
Just a minor change

File: training-app/spec/models/user_spec.rb (hunk 1 above)
Comment (holywyvern) [reply training-app/spec/models/user_spec.rb:6]: While I agree its best for readability, Rubocop complains if I actually do that

**Review by juanisierra — APPROVED** [review]
Well done :+1:

**Pull request merged**